Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -1368,6 +1368,99 @@ final class AppDelegateViewModelTests: TestCase {
     )
   }
 
+  func testVisitorCookies_ApplicationWillEnterForeground() {
+    let existingCookie = HTTPCookie(
+      properties: [
+        .name: "existing-cookie",
+        .value: "existing-cookie-value",
+        .domain: AppEnvironment.current.apiService.serverConfig.apiBaseUrl.host as Any,
+        .path: "/",
+        .version: 0,
+        .expires: Date.distantFuture,
+        .secure: true
+      ]
+    )
+
+    AppEnvironment.current.cookieStorage.setCookie(existingCookie!)
+
+    self.vm.inputs.applicationWillEnterForeground()
+
+    XCTAssertEqual(
+      ["existing-cookie", "vis", "vis"],
+      AppEnvironment.current.cookieStorage.cookies!.map { $0.name }.sorted()
+    )
+    XCTAssertEqual(
```

Comment:
Should we add an assertion for the cookie domain? I think we should have two different domains for the two `vis` cookies.

Reply:
Sure, can do!